internal/services/provisionner.go

```diff
@@ -481,15 +481,26 @@ func updateExistingNamespace(project *v12.Project, api v13.CoreV1Interface) erro
 func generateNamespaceLabels(project *v12.Project) (labels map[string]string) {
 
 	defaultLabels := map[string]string{
-		"name":        project.Name,
-		"type":        "customer",
-		"creator":     "kubi",
-		"environment": project.Spec.Environment,
+		"name":                               project.Name,
+		"type":                               "customer",
+		"creator":                            "kubi",
+		"environment":                        project.Spec.Environment,
+		"pod-security.kubernetes.io/enforce": GetPodSecurityStandardName(project.Name),
+		"pod-security.kubernetes.io/warn":    string(utils.Config.PodSecurityAdmissionWarning),
+		"pod-security.kubernetes.io/audit":   string(utils.Config.PodSecurityAdmissionAudit),
 	}
 
 	return utils.Union(defaultLabels, utils.Config.CustomLabels)
 }
 
+func GetPodSecurityStandardName(namespace string) string {
+	if utils.IsInPrivilegedNsList(namespace) {
+		utils.Log.Warn().Msgf("Namespace %v is labeled as privileged", namespace)
+		return utils.PodSecurityPrivileged
```

Comment:
nit: https://github.com/kubernetes/pod-security-admission/blob/c976ce3661b8678781853e7e84c6f8d0e42be500/api/constants.go#L22

Maybe we can use that constant instead, and remove the string utils.PodSecurityPrivileged from defaults.go (as this _is_  a constant, not a default!) .

Yet, I don't think it's a deal breaker, so let's move on.